client/lxd_instances.go

```diff
@@ -1593,6 +1597,24 @@ func (r *ProtocolLXD) CreateInstanceFile(instanceName string, filePath string, a
 		req.Header.Set("X-LXD-write", args.WriteMode)
 	}
 
+	modifyPerm := []string{}
+
+	if args.ModifyExistingUID {
+		modifyPerm = append(modifyPerm, "uid")
+	}
+
+	if args.ModifyExistingGID {
+		modifyPerm = append(modifyPerm, "gid")
+	}
+
+	if args.ModifyExistingMode {
+		modifyPerm = append(modifyPerm, "mode")
+	}
+
+	if len(modifyPerm) != 0 {
```

Comment:
Or do we want to just silently fallback to old behaviour on older servers and not include the header at all?

Reply:
The server ignores unrecognized headers (I believe I tested sending the new header to a server without the logic to handle it). Somehow I missed `CheckExtension` on my first pass trying to figure out how the list of extensions was used.
My feeling is that silently falling back to the old behavior is probably fine, rather than making this a hard error; we could drop a warning on stderr although that could be obnoxious for library users.